app/controllers/books_controller.rb

```diff
@@ -7,6 +7,8 @@ def index
 
   def show
     @reviews = @book.reviews.page(params[:page])
+    view = Rails.cache.fetch("views/#{@book.id}", expires_in: 2.day) { 0 }
+    Rails.cache.write("views/#{@book.id}", view + 1)
```

Comment:
just wonder how you will handle rank view on next day. Because this cache will exist for 2 days and it still has previous data from yesterday 🤔

Reply:
ตอนแรกตั้งใจใช้ Rails.cache.clear ลบอะค่ะ แล้วก็สร้างใหม่ 🥺 กลัวถ้าเก็บ 1 วันแล้วค่ามันจะหายไปก่อนไหมอะคะ